editoast/src/core/simulation.rs

```diff
@@ -623,4 +631,31 @@ mod tests {
             physics_consist.traction_engine.rolling_resistance,
         );
     }
+
+    #[test]
+    fn physics_consist_compute_gamma() {
+        let mut physics_consist = create_physics_consist();
+        physics_consist.traction_engine.const_gamma = 5.0;
+
+        assert_eq!(
+            Some(physics_consist.compute_const_gamma()),
+            physics_consist
+                .towed_rolling_stock
+                .clone()
+                .map(|t| t.const_gamma)
+        );
+
+        physics_consist.traction_engine.const_gamma = 0.5;
+
+        assert_eq!(
+            physics_consist.compute_const_gamma(),
+            physics_consist.traction_engine.const_gamma,
+        );
+
+        physics_consist.towed_rolling_stock = None;
+        assert_eq!(
+            physics_consist.compute_const_gamma(),
+            physics_consist.traction_engine.const_gamma,
```

Comment:
Are you gonna catch the fact that removing the towed rolling stock isn't taken into account anymore if at the previous step the traction engin already has the lowest gamma ? I feel like the order you change and assert things matter here and it would be better to :
- start with a situation where the towed rolling stock gamma is the lowest
- remove the towed rs
- assert that the gamma of the consist has raised.